src/components/atoms/LongButton.vue

```diff
@@ -0,0 +1,18 @@
+<template>    
```

Comment:
Instead of calling this component "LongButton", I think it would be a good idea giving it a more meaningful name like "ProvideFeedbackButton" or something along the lines.

Reply:
I didn't see this before I merged the branch, but I think that a feedback section would only make sense to add if there was more than one feedback element.

Reply:
That's fair, but the section wouldn't just have a button though. 

For correct document structure under WCAG you would need something like:

```
<main>
  ...
  <section>
    <h2>Feedback section</h2>
    <button></button>
  </section>
</main>
```

Otherwise without that `<section>` element the button would fall under the last heading in main (in this case the last heading might be the conversation window) and the button doesn't below to that document section.

Reply:
That makes sense, but given that it isn't included in the design as of now, it might be best to implement this after a design discussion with Thomas? Though on the other hand it seems a bit too much overhead for just adding a heading and section tag. What do you think?